ee/modules/110-istio/hooks/dataplane_metadata_exporter.go

```diff
@@ -196,11 +198,18 @@ func dataplaneMetadataExporter(input *go_hook.HookInput) error {
 		}
 	}
 
+	ignored := 0
```

Comment:
NIT: You can avoid converting metrics to float64 by adding a type.

```go
var ignored, withoutSidecar, total, worked float64
```
https://go.dev/play/p/bmg0m-JnBI6

Reply:
done